Cargo.toml

```diff
@@ -18,7 +18,7 @@ name = "bin_fixture"
 predicates = { version = "1.0", default-features = false, features = ["difference"] }
 predicates-core = "1.0"
 predicates-tree = "1.0"
-escargot = "0.3"
+escargot = "0.4"
```

Comment:
I've been working towards `assert_cmd` being 1.0.  The current quiet period is me awaiting feedback.  Escargot is further from 1.0.  It is a lot closer now after my recent PRs but I still want to get some more practical usage out of it first (like with `stager` / `cargo-tarball`).

Even once escargot goes 1.0, I expected to not have much resistance to creating a 2.0, etc because I was not expecting it to be in APIs of other crates.  Putting it in another crate's API makes me want to make sure the API is even more solid and could delay 1.0.